03-configure.sh

```diff
@@ -146,3 +146,20 @@ chmod +x /etc/init.d/leds-off
 # Shut all the LEDs down so this takes effect immediately
 /etc/init.d/leds-off enable
 /etc/init.d/leds-off start
+
+# Enable OPKG packages
+ echo "
+  arch all 100
+  arch brcm63xx 200
+  arch brcm63xx-tch 300
+ " > /etc/opkg.conf 
```

Comment:
`/etc/opkg.conf` already has these lines present - at least in my install.  Also, using the redirection operator `>` has the effect of killing the rest of the file, which means the existing opkg config would be lost.